konf/site.yaml

```
@@ -1,4 +1,4 @@
-domain: releases.demo.haus
+domain: webteam.canonical.com
 
 image: prod-comms.docker-registry.canonical.com/releases.demo.haus
```

At this point we can wait until we have staging and the staging job can create the image with the right name:

```suggestion
image: prod-comms.docker-registry.canonical.com/webteam.canonical.com
```